velox/exec/HashProbe.cpp

```diff
@@ -261,13 +514,31 @@ void HashProbe::addInput(RowVectorPtr input) {
     return;
   }
 
+  if (needSpillInput()) {
+    spillInput(input_);
+  }
+
+  // Check if all the probe input rows have been spilled.
```

Comment:
nit: perhaps, move this inside of the 'if (needSpillInput())' block.

Reply:
Good point!